src/main.rs

```diff
@@ -108,6 +110,7 @@ fn main() {
                 .arg(&keymap_arg)
                 .arg(&jobs_arg),
         )
+        .subcommand(SubCommand::with_name("time").about("Prints time in milliseconds"))
```

Comment:
Since `starship time` is sort-of an implementation detail, I think it would be best if we hid it from `starship --help` with the following Clap setting:

```suggestion
        .subcommand(SubCommand::with_name("time").about("Prints time in milliseconds").settings(&[AppSettings::Hidden]))
```

Reply:
Done! thanks.